docker/package/tezos_setup_wizard.py

```diff
@@ -98,9 +98,9 @@ def is_full_snapshot(import_mode):
     id="network",
     prompt="Which Tezos network would you like to use?\nCurrently supported:",
     help="The selected network will be used to set up all required services.\n"
-    "The currently supported protocol is 013-PtJakart (used on jakartanet, ghostnet and mainnet)\n"
+    "The currently supported protocol is 013-PtJakart (used on kathmandunet, ghostnet and mainnet)\n"
     "and 014-PtKathma (used on kathmandunet)"
-    "Keep in mind that you must select the test network (e.g. jakartanet)\n"
+    "Keep in mind that you must select the test network (e.g. kathmandunet)\n"
     "if you plan on baking with a faucet JSON file.\n",
     options=networks,
```

Comment:
Another thing that happens in this wizard are `proc_call`s that call tezos binaries.

This too could work due to the symlinks, but we should probably change it calling octez names instead